includes/class-wsuwp-graduate-degree-programs.php

```diff
@@ -75,6 +75,16 @@ class WSUWP_Graduate_Degree_Programs {
 			'type' => 'string',
 			'sanitize_callback' => 'esc_url_raw',
 		),
+		'gsdp_deadlines' => array(
+			'description' => 'Deadlines',
+			'type' => 'deadlines',
+			'sanitize_callback' => 'WSUWP_Graduate_Degree_Programs::sanitize_deadlines',
+		),
+		'gsdp_requirements' => array(
+			'description' => 'Requirements',
+			'type' => 'requirements',
+			'sanitize_callback' => 'WSUWP_Graduate_Degree_Programs::sanitize_requirements',
```

Comment:
There's no accompanying `sanitize_requirements` function :). I'm guessing you'll add one later or reuse `sanitize_deadlines`, depending on the data these fields are meant to collect.